defaults/main.yml

```
@@ -78,7 +78,7 @@ prometheus_scrape_configs:
 prometheus_config_file: 'prometheus.yml.j2'
 
 prometheus_alert_rules_files:
-  - prometheus/rules/*.rules
+  - "prometheus/rules/*.{{ prometheus_rules_files_extension }}"
```

Since this already expects files in a specific directory, why not add only files which are accepted by promethues plus for some time `*.rules` files? This would conform to best practices in file naming (which we shamelessly violated in the beginning) and provide much simpler solution.

```suggestion
  - "prometheus/rules/*.rules"
  - "prometheus/rules/*.yml"
  - "prometheus/rules/*.yaml"
  - "prometheus/rules/*.json"
```